### ISSUE TITLE
Indent rule doesn’t validate statements outside from blocks

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#1796]
The following code should have one error:

```js
    var a = 42;
if (a) {
    b();
}
```

I would expect the following error:
```sh
1:4  error  Expected indentation of 0 characters  indent
```
instead I get no error.

**Comment by nzakas** [comment]
Working on this.
